sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1521,6 +1521,9 @@ static void _send(gnrc_netif_t *netif, gnrc_pktsnip_t *pkt, bool push_back)
         }
         else {
             LOG_ERROR("gnrc_netif: can't queue packet for sending\n");
+            /* If we got here, it means the device was busy and the pkt queue
+             * was full. The packet should be dropped here anyway */
+            gnrc_pktbuf_release(pkt);
```

Comment:
It's unfortunate that we can't report this error to the upper layer

Reply:
We can, actually...

```suggestion
            gnrc_pktbuf_release_error(pkt, EBUSY);
```

e.g.

Reply:
Or
```suggestion
            gnrc_pktbuf_release_error(pkt, ENOMEM);
```

Not sure, which one makes more sense here.

Reply:
ENOMEM makes more sense to me